parsers/smallFileParser/parse.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+'''
+This program is for smallFile, parses the output and puts important numbers in a
+csv file.
+
+usage: parse.py [-h] [-i INPUT] [-o OUTPUT]
+
+SmallFile Parser
+
+optional arguments:
+  -h, --help                 :  show this help message and exit
+  -i INPUT, --input INPUT    :  smallfile output [path] file name
+  -o OUTPUT, --output OUTPUT :  export parsed result in a file (csv) of the name
+'''
+import re
+import argparse
+import csv
+import os.path
+
+def shred_digit(line):
+    '''
+    This function fetches numbers from a line
+    '''
+    return re.findall(r'\d+.\d+', line)
+
+def find_pattern(file_name):
+    '''
+    This function is to get numbers by filtering pattern
+    '''
+    attr = []
+    with open(file_name) as fobj:
+        data_file = fobj.read().split("\n")
+        for line in data_file:
```

Comment:
Have you looked at whether smallfile already outputs CSV and if you can use that instead?

Reply:
With tie option --output-json smallfile will output JSON data, this maybe more useful from a parsing standpoint.

smallfile does output csv, but that is more for when we give --response-times option, and records response times for each operation. Noting that here. Also, we possibly do not need that.

I would prefer parsing JSON in this case, rather than CLI output.